.github/workflows/esti.yaml

```diff
@@ -964,3 +964,57 @@ jobs:
         if: ${{ failure() }}
         continue-on-error: true
         run: docker-compose -f esti/ops/docker-compose.yaml logs --tail=1000 lakefs
+
+  python-wrapper:
```

Comment:
I just conformed to the rest of our client tests.
We can add it to a different workflow if it's an issue.
Regarding local - since all our client checks are done primarily on S3 I kept it this way. 
I prefer testing on a "production" configuration

Reply:
Will not block using actual lakefs for testing, just think that for wrapper library it will be faster and safer to test the actual expectation which is what API we use in the client library.

About testing with lakeFS - can we use `services` to run lakefs? (like we do in esti) ugc test? It will keep it simple and log the lakeFS output at the end.